test/support/route.go

```diff
@@ -36,3 +40,45 @@ func GetRoute(t Test, namespace, name string) *routev1.Route {
 	t.T().Helper()
 	return Route(t, namespace, name)(t)
 }
+
+func ExposeService(t Test, namespace string, serviceName string, routeName string, servicePort string) *routev1.Route {
```

Comment:
A nitpick: 
```suggestion
func ExposeService(t Test, namespace string, serviceName, servicePort, routeName string) *routev1.Route {
```
To keep service related properties together

Reply:
Also makes me think, do we need to specify route name? Is there any case when caller would need this info?
What about using service name as route name, possibly with some suffix? That way we reasonably assure that Route name is unique.